README.md

```diff
@@ -1,5 +1,22 @@
-# Morpho Blue IRM
+# Morpho Blue IRMs
+
+Some interest rate models for Morpho Blue:
+- [AdaptiveCurveIRM](./src/AdaptiveCurveIrm.sol)
+
+## Resources
+
+- AdaptiveCurveIRM: [docs](https://www.notion.so/morpho-labs/Morpho-Blue-Documentation-Hub-External-00ff8194791045deb522821be46abbdc?pvs=4#d8269074bfd649009f28625a9caa38ea), [announcement article](https://morpho.mirror.xyz/aaUjIF85aIi5RT6-pLhVWBzuiCpOb4BV03OYNts2BHQ).
 
 ## Audits
 
-All audits are stored in the [audits](./audits/)' folder.
\ No newline at end of file
+All audits are stored in the [audits](./audits/)' folder.
```

Comment:
```suggestion
All audits are stored in the [audits](audits)' folder.
```

Reply:
I'm not sure implicit relative paths are better 😁

Reply:
Unresolving because we can be consistent about this, there are other places where we use explicit relative paths. Although still not blocking, feel free to resolve again